SECURITY.md

```diff
@@ -0,0 +1,11 @@
+# Security Policy
+
+Eclipse OpenJ9 follows the [Eclipse Vulnerability Reporting Policy](https://www.eclipse.org/security/policy.php). Vulnerabilities are tracked by the Eclipse OpenJ9 project leads, or the Eclipse security team in cooperation with the OpenJ9 project leads. Fixing vulnerabilities is taken care of by the OpenJ9 project committers.
+
+## Supported Versions
+
+Eclipse OpenJ9 supports security updates only in upcoming releases.
+
+## Reporting a Vulnerability
+
+We recommend that in case of suspected vulnerabilities you do not use the OpenJ9 public issue tracker, but instead contact an Eclipse OpenJ9 project lead via the [OpenJ9 Slack](https://openj9.slack.com/) (join [here](https://join.slack.com/t/openj9/shared_invite/enQtNDU4MDI4Mjk0MTk2LWVhNTMzMGY1N2JkODQ1OWE0NTNmZjM4ZDcxOTBiMjk3NGFjM2U0ZDNhMmY0MDZlNzU0ZjAyNzQ1ODlmYjg3MjA)) and a private channel will be created for the discussion. Or contact the Eclipse Security Team via security@eclipse.org.
```

Comment:
I think this would read better if the paragraph instead began
> In case of suspected vulnerabilities, we recommend that you do not use the OpenJ9 public issue tracker, ...